solidity/solc-update-0.8.15-to-0.8.25.md

```diff
@@ -30,12 +30,8 @@
 
 ## Benefits to the OP Stack
 
-Explain specific feature(s) in the new version that would benefit the OP Stack codebase, with a
-clear explanation of their potential impact and usefulness.
-
 #### DevX
 
-- Allow named parameters in mapping types (>=0.8.18): Limits error when reading mappings of high dimensions when called using named parameters.
```

Comment:
why was this removed?

Reply:
Because its actually not possible. https://github.com/ethereum-optimism/design-docs/pull/106#discussion_r1786445376, https://github.com/ethereum-optimism/design-docs/pull/106#discussion_r1786462875